.gitignore

```diff
@@ -25,3 +25,4 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+echo "coverage" >> .gitignore
```

Comment:
Echo is a linux command that repeats whatever you tell it to. In this case, it will repeat the word `coverage` but add it to the `.gitignore` file.

That command is a simpler way of going to your `.gitignore` file and adding the word "coverage" a the bottom. I should be run from your terminal on your project root.

Reply:
Not related to this PR, move it to the relevant one